donkeycar/parts/telemetry.py

```diff
@@ -38,7 +42,20 @@ def __init__(self, cfg, default_inputs=None, default_types=None):
         self._mqtt_client.connect(self._mqtt_broker, cfg.TELEMETRY_MQTT_BROKER_PORT)
         self._mqtt_client.loop_start()
         self._on = True
-        print(f"Telemetry MQTT server connected (publishing: {', '.join(self._default_inputs)}")
+        if cfg.TELEMETRY_LOGGING_ENABLE:
+            self.setLevel(logging.getLevelName(cfg.TELEMETRY_LOGGING_LEVEL))
+            self.setFormatter(logging.Formatter(cfg.TELEMETRY_LOGGING_FORMAT))
+            logger.addHandler(self)
+
+    def add_step_inputs(self, inputs, types):
+   
+        # Add inputs if supported and not yet registered
+        for ind in range(0, len(inputs or [])):
```

Comment:
probably better to check `inputs` before the `for` statement

Reply:
To check how? Not exactly understand the comment. 

(I changed the way it worked before to reduce the code in `manage.py` and make it more readable.)

Reply:
You only want to run this loop if `inputs` is not empty, i.e. `range(0, len([]))` is empty which is a bit convoluted way of saying `if inputs: .... else:...` or `if not inputs: return ...`

Reply:
I can add a check if you insist (personally I prefer to reduce the code if not necessary). Please let me know